pkg/windows/windows.go

```diff
@@ -428,14 +419,14 @@ func (vm *windows) ensureHostName() error {
 func (vm *windows) isHostNameChangeNeeded() (bool, error) {
 	out, err := vm.Run("hostname", true)
 	if err != nil {
-		return false, errors.Wrapf(err, "error getting the host name for %s with stdout %s", vm.ID(), out)
+		return false, errors.Wrapf(err, "error getting the host name, with stdout %s", out)
 	}
-	return !strings.Contains(out, vm.hostName), nil
+	return !strings.Contains(out, vm.machineName), nil
 }
 
 // changeHostName changes the hostName of the Windows VM to match with the machine name
 func (vm *windows) changeHostName() error {
-	changeHostNameCommand := "Rename-Computer -NewName " + vm.hostName + " -Force -Restart"
+	changeHostNameCommand := "Rename-Computer -NewName " + vm.machineName + " -Force -Restart"
```

Comment:
Consider building the resulting `string` with a string formatting approach,  `fmt.Sprintf()` maybe. 

Should we entertain a default locale for  `vm.machineName`?